Alamofire.xcodeproj/project.pbxproj

```diff
@@ -1659,11 +1659,6 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				OTHER_LDFLAGS = (
-					"-framework",
-					CFNetwork,
```

Comment:
We still need the CFNetwork linking, please add it back.

Reply:
Hello @jshier 
`OTHER_LDFLAGS = ("-framework", CFNetwork)` is set for Alamofire project, hence Alamofire iOS, Alamofire macOS, Alamofire tvOS and Alamofire watchOS also have these parameters.

Reply:
Ah, you're right, thanks.